### PR TITLE
GBE-865: Restored Coordinator box to ReviewVendorView (plus test)

### PR DESCRIPTION
**Pull request by jonkiparsky**

### CODE REVIEW

File: expo/gbe/views/review_vendor_view.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class ReviewVendorView(ReviewBidView):
     reviewer_permissions = ('Vendor Reviewers',)
-    coordinator_permissions = ('Vendor Coordinator')
```
Comment (bethlakshmi) [inline expo/gbe/views/review_vendor_view.py]: Yep... when you said comma, this is where I went to in my head.
Aw man... ick!

**Comment by bethlakshmi** [comment]
Approved.
- ran pepdiff - no findings
- ran unit tests - all good
- looked at it - love it
- manual test - vendor coordintor now sees bid coordinator box.

**Comment by BenignCremator** [comment]
Just pulled this down and set ip up. Functional test looks good. Progressing onto code review.

**Comment by BenignCremator** [comment]
Love the code review.  Trying py.test with and without the ForeignKey fix I just found.

**Comment by BenignCremator** [comment]
Fix not working on unit test, but they worked for @bethlakshmi, so approving.

**Pull request merged**